torch/_torch_docs.py

```diff
@@ -7102,6 +7108,9 @@ def merge_dicts(*dicts):
     tensor([[ 1.0000,  0.3139],
             [ 0.3898,  1.0000],
             [ 0.0478,  1.0000]])
+    >>> torch.where(torch.randn(2,2) > 0.5, torch.randn(2,2).to(torch.double), 0.)
```

Comment:
Just construct the condition tensor, x, and y directly so the reader can see their values. Otherwise it's hard to understand what the function does because they don't know what the condition or x tensors contained.

Reply:
(The existing documentation prints the random tensor it creates to avoid this issue)

Reply:
Makes sense. Will do that.